src/priorityQueue.js

```diff
@@ -0,0 +1,72 @@
+export class Element {
+  constructor(value, priorityInt) {
+    this.data = value
+    this.priority = priorityInt
+  }
+}
+
+export default class PriorityQueue {
+  constructor() {
+    this.elements = []
+    this.size = 0
+  }
+
+  enqueue(value, priorityInt) {
+    const element = new Element(value, priorityInt)
+    this.elements.push(element)
+    this.size += 1
+  }
+
+  front = () => {
+    if(this.size === 0) return null
```

Comment:
Consider avoiding the use of inline control flow, to increase readability and maintainability.  I favor the use of explicit braces to avoid possible subtle dangling `if/else`, as well.